### PR TITLE
Updated mainnet contract info

### PR DESCRIPTION
**Pull request by psy2848048**

### CODE REVIEW

File: content/docs/howto/token.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ Except any function of your token itself contains more than asset, we recommend
 
 ## How to mint
 
-### 1. Using deployed token (recommended)
+### 1. Using deployed token factory (recommended)
 
-The standard CW20 token is already stored in Terra network as code ID `3`.\
-The code is wrapped based on cw20 [v0.8.0](https://docs.rs/cw20/0.8.0/cw20/index.html) \
+The standard CW20 token is already stored in Terra 2.0 network as code ID `4`, and Terra classic as `3`.\
+The code is wrapped based on cw20 [v0.11.0](https://docs.rs/cw20/0.11.0/cw20/index.html) \
```
Comment (JoowonYun) [inline content/docs/howto/token.md:30]: We used 0.13.2

Comment (psy2848048) [reply content/docs/howto/token.md]: https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/35/commits/3af6325c4cbb4873cc7f33efa9a0b0a00d3efb9f

File: content/docs/reference/router.md [hunk 2]
```diff
@@ -160,3 +160,184 @@ Number of ANC needed to get 10Luna <br>
...
+        }
+    })
+}
+```
+
+## Query
+
+
+### Simulate swap operations
+Number of KRT that can be get through 10 mABNB <br>
```
Comment (JoowonYun) [inline content/docs/reference/router.md:264]: KRT -> Luna

mABNB -> TNT

File: content/docs/reference/router.md [hunk 3]
```diff
@@ -160,3 +160,184 @@ Number of ANC needed to get 10Luna <br>
...
 ex) https://bombay-fcd.terra.dev/wasm/contracts/terra1c58wrdkyc0ynvvxcv834kz65nfsxmw2w0pwusq/store?query_msg={%22reverse_simulate_swap_operations%22:{%22ask_amount%22:%2210000000%22,%22operations%22:[{%22terra_swap%22:{%22offer_asset_info%22:{%22token%22:{%22contract_addr%22:%22terra1747mad58h0w4y589y3sk84r5efqdev9q4r02pc%22}},%22ask_asset_info%22:{%22native_token%22:{%22denom%22:%22uusd%22}}}},{%22terra_swap%22:{%22offer_asset_info%22:{%22native_token%22:{%22denom%22:%22uusd%22}},%22ask_asset_info%22:{%22native_token%22:{%22denom%22:%22uluna%22}}}}]}}
+
+# Router - Terra 2.0
+
+
+## Transaction
+
+### Execute swap operations
+
+Multi-hop swap operations via `native_swap`, `terra_swap`, `loop` and `astroport`<br />
```
Comment (JoowonYun) [inline content/docs/reference/router.md:171]: only terraswap

File: content/docs/reference/router.md [hunk 4]
```diff
@@ -160,3 +160,184 @@ Number of ANC needed to get 10Luna <br>
...
+               }
+            }
+         }
+      ],
+      "minimum_receive":"1"
+   }
+}
+```
+
+Case 2) The first source token is a CW20 token (ANC => UST => KRT)<br />
```
Comment (JoowonYun) [inline content/docs/reference/router.md:213]: Luna -> DELIGHT -> TNT

Comment (psy2848048) [reply content/docs/reference/router.md]: https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/35/commits/66d588098e7a98f89f5c5d55d83ea443cc9e6789

File: content/docs/reference/router.md (hunk 3 above)
Comment (psy2848048) [reply content/docs/reference/router.md]: https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/35/commits/66d588098e7a98f89f5c5d55d83ea443cc9e6789

File: content/docs/reference/router.md (hunk 2 above)
Comment (psy2848048) [reply content/docs/reference/router.md]: https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/35/commits/66d588098e7a98f89f5c5d55d83ea443cc9e6789

File: content/docs/reference/router.md [hunk 5]
```diff
@@ -160,3 +160,183 @@ Number of ANC needed to get 10Luna <br>
...
+                            "contract_addr":"terra1qnypzwqa03h8vqs0sxjp8hxw0xy5zfwyax26jgnl5k4lw92tjw0scdkrzm"
+                        }
+                    }
+                }
+            }
+        ]
+    }
+}
+```
+ex) https://bombay-fcd.terra.dev/wasm/contracts/terra1c58wrdkyc0ynvvxcv834kz65nfsxmw2w0pwusq/store?query_msg={%22simulate_swap_operations%22:{%22offer_amount%22:%2210000000%22,%22operations%22:[{%22terra_swap%22:{%22offer_asset_info%22:{%22token%22:{%22contract_addr%22:%22terra1747mad58h0w4y589y3sk84r5efqdev9q4r02pc%22}},%22ask_asset_info%22:{%22native_token%22:{%22denom%22:%22uusd%22}}}},{%22native_swap%22:{%22offer_denom%22:%22uusd%22,%22ask_denom%22:%22ukrw%22}}]}}
```
Comment (JoowonYun) [inline content/docs/reference/router.md:302]: https://pisco-lcd.terra.dev/cosmwasm/wasm/v1/contract/terra1xp6xe6uwqrspumrkazdg90876ns4h78yw03vfxghhcy03yexcrcsdaqvc8/smart/ewogICAgInNpbXVsYXRlX3N3YXBfb3BlcmF0aW9ucyI6ewogICAgICAgICJvZmZlcl9hbW91bnQiOiIxMDAwMDAwMCIsCiAgICAgICAgIm9wZXJhdGlvbnMiOlsKICAgICAgICAgICAgewogICAgICAgICAgICAgICAgInRlcnJhX3N3YXAiOnsKICAgICAgICAgICAgICAgICAgICAib2ZmZXJfYXNzZXRfaW5mbyI6ewogICAgICAgICAgICAgICAgICAgICAgICAibmF0aXZlX3Rva2VuIjp7CiAgICAgICAgICAgICAgICAgICAgICAgICAgICAiZGVub20iOiJ1bHVuYSIKICAgICAgICAgICAgICAgICAgICAgICAgfQogICAgICAgICAgICAgICAgICAgIH0sCiAgICAgICAgICAgICAgICAgICAgImFza19hc3NldF9pbmZvIjp7CiAgICAgICAgICAgICAgICAgICAgICAgICJ0b2tlbiI6ewogICAgICAgICAgICAgICAgICAgICAgICAgICAgImNvbnRyYWN0X2FkZHIiOiJ0ZXJyYTFjbDBrdzlheHpwemt3NThzbmo2Y3kwaGZwMHhwOHhoOXR1ZHB3MmV4dnp1dXBuM2ZhZndxcWhqYzI0IgogICAgICAgICAgICAgICAgICAgICAgICB9CiAgICAgICAgICAgICAgICAgICAgfQogICAgICAgICAgICAgICAgfQogICAgICAgICAgICB9LAogICAgICAgICAgICB7CiAgICAgICAgICAgICAgICAidGVycmFfc3dhcCI6ewogICAgICAgICAgICAgICAgICAgICJvZmZlcl9hc3NldF9pbmZvIjp7CiAgICAgICAgICAgICAgICAgICAgICAgICJ0b2tlbiI6ewogICAgICAgICAgICAgICAgICAgICAgICAgICAgImNvbnRyYWN0X2FkZHIiOiJ0ZXJyYTFjbDBrdzlheHpwemt3NThzbmo2Y3kwaGZwMHhwOHhoOXR1ZHB3MmV4dnp1dXBuM2ZhZndxcWhqYzI0IgogICAgICAgICAgICAgICAgICAgICAgICB9CiAgICAgICAgICAgICAgICAgICAgfSwKICAgICAgICAgICAgICAgICAgICAiYXNrX2Fzc2V0X2luZm8iOnsKICAgICAgICAgICAgICAgICAgICAgICAgInRva2VuIjp7CiAgICAgICAgICAgICAgICAgICAgICAgICAgICAiY29udHJhY3RfYWRkciI6InRlcnJhMXFueXB6d3FhMDNoOHZxczBzeGpwOGh4dzB4eTV6Znd5YXgyNmpnbmw1azRsdzkydGp3MHNjZGtyem0iCiAgICAgICAgICAgICAgICAgICAgICAgIH0KICAgICAgICAgICAgICAgICAgICB9CiAgICAgICAgICAgICAgICB9CiAgICAgICAgICAgIH0KICAgICAgICBdCiAgICB9Cn0=

Comment (psy2848048) [reply content/docs/reference/router.md]: https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/35#:~:text=updated%20example%20link-,7c17b80,-Merge%20state

**Review by JoowonYun — APPROVED** [review]
LGTM

**Pull request merged**